README.md

```diff
@@ -101,36 +100,56 @@ Prerequisites:
     export OASIS_GO=go1.13.8
     ```
 
-* [Rust](https://www.rust-lang.org) and the nightly toolchain.
+* [Rust].
 
-  Once you have [`rustup` installed](https://www.rust-lang.org/tools/install),
-  install the nightly with:
+  We follow [Rust upstream's recommendation][rust-upstream-rustup] on using
+  [rustup] to install and manage Rust versions.
 
-  ```
-  rustup install nightly
-  ```
+  _NOTE: [rustup] cannot be installed alongside a distribution packaged [Rust]
+  version. You will need to remove it before you can start using [rustup]._
 
-  Then make it the default version with:
+  Install it by running:
 
   ```
-  rustup default nightly
+  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
```suggestion
  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain nightly
```
if you're feeling terse

Reply:
Thanks for the suggestions. I decided to leave it as is as I would prefer not to set the nightly as a default toolchain for all user's projects.